CanvasPlusPlayground/Model/Intelligence/LLMEvaluator.swift

```diff
@@ -87,10 +87,9 @@ class LLMEvaluator: ObservableObject {
                 thread: thread,
                 systemPrompt: systemPrompt
             )
-            let prompt = modelConfiguration.prepare(prompt: promptHistory)
 
             let promptTokens = await modelContainer.perform { _, tokenizer in
-                tokenizer.encode(text: prompt)
+                tokenizer.encode(text: promptHistory)
```

Comment:
I'm installing the 3b version and it takes a long time to download, is that expected? Perhaps once they trigger installation, they can still use the app and check progress when they want.

Reply:
The app needs to be open for the download to finish. We need to find a way to enforce that, or figure out background downloads.